packages/x-scheduler/src/primitives/use-event-calendar/useEventCalendar.types.ts

```diff
@@ -110,3 +112,31 @@ export interface EventCalendarContextValue {
    */
   instance: EventCalendarInstance;
 }
+
+/**
+ * The scope of a recurring event update.
+ */
+export type RecurringUpdateEventScope = 'following' | 'all' | 'only-this';
+
+/**
+ * Parameters for updating a recurring event.
+ */
+export type UpdateRecurringEventParams = {
```

Comment:
```suggestion
export type UpdateRecurringEventParameters = {
```

To follow the Base UI nomenclature